storage/src/test/java/org/apache/uniffle/storage/common/DefaultStorageMediaProviderTest.java

```diff
@@ -40,4 +40,11 @@ public void testStorageProvider() {
     // invalid uri should also be reported as HDD
     assertEquals(StorageMedia.HDD, provider.getStorageMediaFor("file@xx:///path/to/a"));
   }
+
+  @Test
+  public void getGetDeviceName() {
```

Comment:
I think it's not predictable. Depends on the CI environment.

Reply:
I don't know how to do it. I don't know which path will return rootfs in CI environment.

Reply:
> how about mock the `getDeviceName` method?
> 
> Seems it was declared as static, maybe an instance method is easier to be mocked?

I just tried, but failed. We need to specify the return value for `getStorageMediaFor`, otherwise null will be returned.